packit/config.py

```diff
@@ -32,6 +32,7 @@ def __init__(self):
 
         self._pagure_user_token: str = ""
         self._pagure_fork_token: str = ""
+        self.dry_run: bool = False
```

Comment:
also please enable loading this from the user config and document it in docs/configuration.md

Reply:
yep, added.